content/source/docs/cloud/api/changelog.html.md

```diff
@@ -0,0 +1,20 @@
+---
+page_title: "API changelog - API Docs - Terraform Cloud and Terraform Enterprise"
+layout: "cloud"
+page_id: "api-changelog"
+---
+
+[breaking]: ./api/changelog/breaking.png "Breaking"
+[cloud]: ./api/changelog/cloud.png "Cloud"
+[enterprise]: ./api/changelog/enterprise.png "Enterprise"
+
+# API Changelog
+
+Keep track of changes to the Terraform Cloud and Terraform Enterprise APIs.
+
+### 2021-01-21
+
+* ![cloud] A new Invoices API is available.
```

Comment:
Noting for posterity, not actually a blocker on this: The "empty reference" style (`![cloud][]`) is more compatible with various Markdown renderers than the "absent reference" style. (But it's also fairly find-and-replaceable, so whatever.)

Reply:
Oh, thanks for the tip. I copied this from online somewhere. I'll update to match.